src/configuration/routes/cache.md

```diff
@@ -1,16 +1,35 @@
-# HTTP Cache
+# HTTP cache
 
-## Rationale
+Platform.sh supports HTTP caching at the server level. The cache is only applied to `GET` and `HEAD` requests.
 
-Many hosting solutions are adding an additional layer (Varnish...) to handle caching. Platform.sh supports HTTP caching right at the web server level.
+The cache must be explicitly enabled using the `cache` key in your `.platform/routes.yaml` file. Platform.sh serves the files or requests generated by the application directly, unless the cache is enabled.
 
-By default, the router will respect whatever cache headers are sent by the application (either from your application code or from the `.platform.app.yaml` configuration), but the presence of any cookie headers will cause the cache to be bypassed.
+To decide how to cache a response, Platform.sh will build a cache key depending on several factors and store the response associated with this key. When a request comes with the same cache key, the cached response will be reused.
 
-If you disable caching, Platform.sh serves the files or requests generated by the application directly.
+If you enable caching...
 
-## Reference
+* you can configure cache behaviour for different location blocks in your `.platform.app.yaml`
+* the router will respect whatever cache headers are sent by the application
+* cookies will not bypass the cache unless the `cookie` property is set or the `Set-Cookie` header is present
```

Comment:
I don't understand this point. Suggest:

* the cache key will vary based on the cookies that are configured in `cookies`